src/Stripe.net/Entities/Subscriptions/SubscriptionPendingUpdate.cs

```diff
@@ -0,0 +1,52 @@
+namespace Stripe
+{
+    using System;
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
+    public class SubscriptionPendingUpdate : StripeEntity<SubscriptionPendingUpdate>
+    {
+        /// <summary>
+        /// If the update is applied, determines the date of the first full
+        /// invoice, and, for plans with <c>month</c> or year <c>intervals</c>,
+        /// the day of the month for subsequent invoices.
+        /// </summary>
+        [JsonProperty("billing_cycle_anchor")]
+        [JsonConverter(typeof(DateTimeConverter))]
+        public DateTime? BillingCycleAnchor { get; set; }
+
+        /// <summary>
+        /// The point after which the changes reflected by this update will be
+        /// discarded and no longer applied.
+        /// </summary>
+        [JsonProperty("expires_at")]
+        [JsonConverter(typeof(DateTimeConverter))]
+        public DateTime? ExpiresAt { get; set; }
+
+        /// <summary>
+        /// List of subscription items, each with an attached plan, that will
+        /// be set if the update is applied.
+        /// </summary>
+        [JsonProperty("subscription_items")]
+        public StripeList<SubscriptionItem> SubscriptionItems { get; set; }
```

Comment:
Shouldn't this be a `List` and not a `StripeList`?

Reply:
Yeah I think you're right. This was one of the things I wanted to test deserializing with stripe-mock.